packages/composite-checkout/src/components/checkout-provider.tsx

```diff
@@ -195,8 +195,17 @@ function useCallEventCallbacks( {
 	paymentMethodId: string | null;
 	transactionLastResponse: PaymentProcessorResponseData;
 } ): void {
+	// Ensure that onPaymentComplete is only called once per checkout so that the URL
```

Comment:
This comment only makes sense in the context of calypso checkout, but this npm package might be used for checkout on many platforms in the future (eg: Akismet, Crowdsignal) which have no cookies or URLs. Even within calypso, we use `CheckoutProvider` for other purposes like payment method assignment. I therefore think we should remove this comment.